jaeger/jaeger_install/rhbjaeger-installation.adoc

```diff
@@ -0,0 +1,43 @@
+[id="installing-jaeger"]
+= Installing Jaeger
+include::modules/jaeger-document-attributes.adoc[]
+:context: jaeger-install
+
+toc::[]
+
+You can install Jaeger on {product-title} in either of two ways:
```

Comment:
s/You can install Jaeger on {product-title} in either of two ways:/You can install Jaeger on {product-title} two ways:/